ReactNativeClient/lib/models/Setting.js

```diff
@@ -164,6 +164,8 @@ class Setting extends BaseModel {
 
 			'startMinimized': { value: false, type: Setting.TYPE_BOOL, section:'application', public: true, appTypes: ['desktop'], label: () => _('Start application minimised in the tray icon') },
 
+			'showTrayIconMinimisedMessage': {value: true, type: Setting.TYPE_BOOL, section:'application', public: true, appTypes: ['desktop'], label: () => _('Remind that the application is being minimised to the tray icon and not actually closed') },
```

Comment:
It would be a hidden setting since it can only be set programmatically, so it should have `public: false`, and no "section" and no "label" key. And let's name it "showTrayIconMinimizedMessage" for consistency since all the settings use US English.